### DISCUSSION TITLE
Is there a way to highlight invalid drop zones?

### DISCUSSION THREAD

**Discussion by jcamejo** [GrapesJS/grapesjs#3602] [Q&A]
Just the opposite of highlight possible drop options, I want to be able to highlight invalid elements.

Is this possible?

Thanks!

**Comment by artf** [comment]
Hi Juan, unfortunately, nothing direct but I guess it might be possible to retrieve the difference between the valid nodes and the rest of the document.

**Reply by jcamejo** [reply]
Thanks! I am trying to solve it that way

**Comment by Agobin** [comment]
If you're able to identify the invalid elements, you can hook into the "canvas:dragenter" or "sorter:drag:start" events and apply a class to all invalid elements. In this class (e.g .invalid), you can define how you want invalid elements to look like.

And finally, when the dragging stops you can hook into "canvas:dragend" or "sorter:drag:end" and remove the class you added before from all invalid elements.
